src/components/filters/useLastSeenFilter.js

```diff
@@ -93,7 +93,7 @@ export const useLastSeenFilter = (
         const todaysDate = moment().endOf('day');
         const selectedFromDate = moment(date).startOf('day');
 
-        if (!containsSpecialChars(date) && selectedFromDate < todaysDate) {
+        if (!containsSpecialChars(date) && selectedFromDate < todaysDate && date.length > 9 && selectedFromDate > oldestDate) {
```

Comment:
Is there something that dictates what the `date.length` has to be, so we don't need to use a hardcoded `9` here?

Reply:
its more like a 'standard; length because with the format used in the component, MM-DD-YYYY, you have to have at least 10 characters in the field, similar to a phone number

Reply:
I think in this case we could have it as a descriptive constant like `DEFAULT_DATE_LENGTH`.